sh/build/helm-template.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM alpine/helm:3.1.1
+
+RUN apk add bash curl git
+RUN apk update
+
+RUN curl -fsSL -o /usr/local/bin/kpt https://storage.googleapis.com/kpt-dev/latest/linux_amd64/kpt
+RUN chmod +x /usr/local/bin/kpt
+ENV PATH /usr/local/bin:$PATH
```

Comment:
why is this needed? What's the default value of PATH

Reply:
It's a best practice from Docker docs, "use ENV to update the PATH environment variable for the software your container installs"

Reply:
But you're using a well-known bin directory, not a user-defined one.